src/mozillavpn.cpp

```diff
@@ -429,6 +439,9 @@ void MozillaVPN::maybeStateMain() {
   maybeRegenerateDeviceKey();
 
   if (state() != StateUpdateRequired) {
+    // All users who get to StateMain (home screen) should never see onboarding
+    // in the future
+    settingsHolder->setOnboardingCompleted(true);
```

Comment:
Just wondering, is there a way to easily test the new onboarding flow without resetting the app?

Reply:
you can sign out and use the inspector command `set_setting onboardingCompleted false`